code-of-conduct.md

```diff
@@ -66,42 +56,39 @@ that are not aligned to this Code of Conduct, or to ban temporarily or
 permanently any community member for other behaviors that they deem
 inappropriate, threatening, offensive, or harmful.
 
-## Enforcement
-
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported in the following ways. You may send an open message from any channel
-to all Community Helpers by tagging @community helpers. You may send an open
-message from any channel or a direct message to @kattni#1507, @tannewt#4653,
-@Dan Halbert#1614, @cater#2442, sommersoft#0222, or @Andon#8175. You may
-also send an email to <support@adafruit.com>. All complaints will
-be reviewed and investigated, and will result in a response that is deemed
-necessary and appropriate to the circumstances. The moderation team is
-obligated to maintain confidentiality with regard to the reporter of an
-incident. Be aware, if you send a message in an open channel, your report will
-not be anonymous. In situations where the issue is particularly egregious,
-requires immediate action, and no moderators are available, you can also report
-the message directly to Discord.
-
-Further details of specific enforcement policies are outlined in the
-Moderation section.
-
 ## Moderation
 
-These are the policies for upholding our community’s standards of conduct.
+Instances of behaviors that violate the Adafruit CircuitPython Code of Conduct
+may be reported by any member of the community. Community members are
+encouraged to report these situations, including situations they witness
+involving other community members.
+
+You may report in the following ways:
+
+In any situation, you may send an email to <support@adafruit.com>.
 
-1. Messages, comments, nicknames or avatar images that violate the Adafruit
-CircuitPython standards of conduct, including anything hateful, hurtful,
-oppressive, or exclusionary, are not allowed.
-2. Messages, comments, nicknames or avatar images that moderators find
-inappropriate, whether listed in the code of conduct or not, are also not
-allowed.
+On the Adafruit Discord, you may send an open message from any channel
+to all Community Helpers by tagging @community helpers. You may also send an
+open message from any channel or a direct message to @kattni#1507, 
+@tannewt#4653, @Dan Halbert#1614, @cater#2442, @sommersoft#0222, or @Andon#8175.
+
+In situations on Discord where the issue is particularly egregious, requires
+immediate action, and no moderators are available, you can also report the
+message directly to Discord.
+
+These are the steps for upholding our community’s standards of conduct.
```

Comment:
This section below still feels out of place to me because its relatively long. I think it'd be better as a pinned post for moderators to follow or separate "moderator how-to" doc that this can link to.

I think the key takeaway for community folks is that for less egregious actions a moderator will reach out privately to involved parties, discuss the issue and make a decision based on the response of folks and after consulting with at least one other moderator as a sanity check. Its important for everyone involved to be open to feedback and be willing to change their behavior.

Reply:
I agree that that is the takeaway, however I don't agree that the section should be removed. I can definitely work through it to tighten it up a bit.

The outline of this section is pulled from Rust's CoC. I know we don't want to get too specific on what is and isn't allowed because then we're limiting our scope, but I feel like we should be more specific when it comes to the moderation steps. We're not limiting ourselves with the way this is written, and we're making it entirely clear what will happen if someone violates the CoC. Right now, we don't say to people on their first warning "This is your first warning, depending on how you act you might get a couple more". We simply engage the individual and then keep an eye on them moving forward. I feel like it's an important thing to include so people know the process. Codes of Conduct aren't necessarily there for the people who are following them, they're there for the people who aren't. And those are the people who would need to have the moderation steps laid out for them.